### ISSUE TITLE
jQuery is missing in bower.json when bootstrap 4 is selected

### ISSUE THREAD

**Issue by LukeSkywalker92** [yeoman/generator-webapp#718]
When I run `yo webapp` (3.0.1) and select bootstrap 4 jQuery is not added to bower.json. This is the bower.json generated by webapp:

```
{
  "name": "test",
  "private": true,
  "dependencies": {
    "bootstrap": "~4.0.0-alpha.6",
    "modernizr": "~2.8.1"
  },
  "devDependencies": {
    "chai": "^4.1.2",
    "mocha": "^5.1.1"
  }
}
```

**Comment by vincenttsao14** [comment]
Read through the fix above but I think I'm still having issues with this? Installed generator-webapp today and tried to scaffold an app with Bootstrap 4.

? Which additional features would you like to include? Sass, Bootstrap, Modernizr
? Which version of Bootstrap would you like to include? Bootstrap 4
? Choose your style of DSL BDD

Successful build but bower.json is still missing jQuery and Popper.js, nor were they installed in the bower_components folder. I tried updating generator-webapp and bower but no dice. Bower json below:

```
{
  "name": "yotest4",
  "private": true,
  "dependencies": {
    "bootstrap": "~4.0.0-alpha.6",
    "modernizr": "~2.8.1"
  },
  "devDependencies": {
    "chai": "^4.1.2",
    "mocha": "^5.2.0"
  }
}
```

**Comment by djmtype** [comment]
Also, when Bootstrap 4 is included as part of the build, it's no longer being called inside the `main.scss` file.

Requires: 

```
// bower:scss
@import "bower_components/bootstrap/scss/bootstrap.scss";
// endbower
```

**Comment by silvenon** [comment]
When we remove Bower and wiredep, you'll unfortunately have to add these imports manually.

**Comment by djmtype** [comment]
@silvenon Gotcha.

**Comment by silvenon** [comment]
Released [v4.0.0-0](https://github.com/yeoman/generator-webapp/releases/tag/v4.0.0-0). Let me know if it's clear what to do.
